packages/cli/src/utils/object-to-error.ts

```diff
@@ -34,15 +36,10 @@ export function objectToError(errorObject: unknown, workflow: Workflow): Error {
 			error = new Error(errorObject.message);
 		}
 
-		if ('description' in errorObject) {
-			// @ts-expect-error Error descriptions are surfaced by the UI but
-			// not all backend errors account for this property yet.
-			error.description = errorObject.description as string;
-		}
-
-		if ('stack' in errorObject) {
-			// If there's a 'stack' property, set it on the new Error instance.
-			error.stack = errorObject.stack as string;
+		for (const field of transientErrorProperties) {
+			if (field in errorObject) {
+				(error as unknown as Record<string, unknown>)[field] = errorObject[field];
```

Comment:
I don't think the types-jugglery is worth the effort here
```suggestion
				// @ts-expect-error Not all errors contain these properties
				error[field] = errorObject[field];
```

Reply:
should we maybe add checks to make sure we copy over only truthy values here?

Reply:
This does imply dropping a `description` that is just an empty string, which is technically a change in behavior? I wouldn't know if any code relies on that.

Reply:
I still don't think the types here provide any more value than disabling type-checking altogether. but we don't need to block this PR on that.

Reply:
The main advantage here is that e.g. `[field]` still gets typechecked to be a string, and `errorObject` remains typed as well